browser/src/control/jsdialog/Widget.TreeView.ts

```diff
@@ -499,13 +499,32 @@ class TreeViewControl {
 			img.src = image;
 			img.alt = text;
 		} else {
-			const cell = L.DomUtil.create(
-				'span',
-				builder.options.cssClass +
-					` ui-treeview-cell-text ui-treeview-cell-text-content ui-treeview-${entry.row}-${index}`,
-				parent,
-			);
-			cell.innerText = text;
+			let cell;
+			if (this.isPageDivider(entry)) {
+				cell = L.DomUtil.create(
+					'span',
+					builder.options.cssClass +
+						` ui-treeview-cell-text ui-treeview-cell-text-content ui-treeview-${entry.row}-${index} page-divider`,
+					parent,
+				);
+				cell.innerText = entry.text.substring(4, entry.text.length - 4);
```

Comment:
Hardcoded 4, we will forgot what it was.
Could you create funtion for it: `getPageEntryText(originalText)` ?
with comment what is in that 4 char string